File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/ResponseGeocoding.scala
package org.openapitools.server.model


/**
 * @param `type`  for example: ''null''
 * @param features  for example: ''null''
*/
final case class ResponseGeocoding (
  `type`: String,
  features: Seq[ResponseGeocodingGeoJsonFeature]
)

File: kotlin-spring/src/main/kotlin/org/openapitools/model/ResponseGeocoding.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.ResponseGeocodingGeoJsonFeature
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param type 
 * @param features 
 */
data class ResponseGeocoding(

    @field:JsonProperty("type", required = true) val type: kotlin.String,

    @field:Valid
    @field:JsonProperty("features", required = true) val features: kotlin.collections.List<ResponseGeocodingGeoJsonFeature>
) {

}

